cms/static/js/views/settings/grader.js

```diff
@@ -11,15 +11,25 @@ var GraderView = ValidatingView.extend({
         "click .remove-grading-data" : "deleteModel",
         // would love to move to a general superclass, but event hashes don't inherit in backbone :-(
         'focus :input' : "inputFocus",
-        'blur :input' : "inputUnfocus"
+        'blur :input' : "inputUnfocus",
+        'click #field-course-grading-assignment-passing-grade-enabled' : "togglePassingGrade"
```

Comment:
`#field-course-grading-assignment-passing-grade-enabled` it's very complicated `id`.